apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -218,6 +218,7 @@ class ManageStudentsTable extends Component {
               {showShareColumn &&
                 <ManageStudentsActionsHeaderCell
                   editAll={this.props.editAll}
+                  isShareColumnVisible={this.props.showSharingColumn}
```

Comment:
small nit I should've noticed this earlier. might be best to do in a follow up PR.
`showSharingColumn` sounds like an action/function (because it sounds like a verb), which is confusing, because we actually have actions that do show the sharing column. Ideally, the redux state also uses the name `isShareColumnVisible`, which is more obviously a boolean value.

Reply:
Good idea. I'll clean up in a separate PR.